### ISSUE TITLE
[ Feature Request ] Copy magnet link option in contextual menu before opening episode stream

### ISSUE THREAD

**Issue by mpcabete** [Stremio/stremio-features#310]
It would be nice if it was possible to right-click in the episode from a torrent and copy it's magnet link before opening it. For slow torrents it is only possible to get the torrent link after it is finished with the initial loading.

**Comment by radenko98** [comment]
Would love this option!

**Comment by ckashila** [comment]
would love this!

**Comment by keneedy** [comment]
yes, please! its pretty annoying when you have a low seed torrent and can't open it to get the magnet link. This would also contribute to more seeders for low sources as people would download it and seed, rather than watch-only and forget about it.

**Comment by MBaltz** [comment]
Please, we need that ;-;

**Comment by vigna** [comment]
I was going to ask for the same thing—Stremio is great also to look for torrents, but sometimes your connection is so slow you cannot get to the viewing screen. Then you're stuck as the magnet menu is not available before.

**Comment by IvanDL95** [comment]
I'd really like this. Torrenting in my country is not nearly as popular as it should be and it's preventing me to watch some shows.

**Comment by LeonardoMacielDeschamps** [comment]
It would be fantastic if we had this functionality.

**Comment by dsamblas** [comment]
yes waiting for this feature ,  would be awesome

**Comment by kKaskak** [comment]
yeah implemented, web.stremio.com v5.0.0-beta.33
